queue_job/migrations/13.0.3.6.0/pre-migration.py

```diff
@@ -0,0 +1,35 @@
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
```

Comment:
ah crap, we already have 13.0.3.6.0 in https://github.com/OCA/queue/blob/263fbaac33a5b559c77a1bb08b0616bb42c78d2f/queue_job/__manifest__.py#L6

@simahawk could you bump the migration directory?

Reply:
ops, missed it. Done!

Reply:
Thanks :green_heart: